daemon/daemon_windows.go

```diff
@@ -470,6 +470,10 @@ func (daemon *Daemon) stats(c *container.Container) (*types.StatsJSON, error) {
 // daemon to run in. This is only applicable on Windows
 func (daemon *Daemon) setDefaultIsolation() error {
 	daemon.defaultIsolation = containertypes.Isolation("process")
+	// On client SKUs, default to Hyper-V
+	if system.IsWindowsClient() {
+		daemon.defaultIsolation = containertypes.Isolation("hyperv")
```

Comment:
Heh, I think @LK4D4 usually doesn't like "else" if the first branch does a *return*; in this case it's assigning a value twice (overriding the first), so it felt cleaner to do if/else, but it's just a nit